recipes/metaclock/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "metaclock" %}
+{% set version = "1.0.0" %}
+
+
+
+package:
+  name:  "{{ name }}"
+  version:  "{{ version }}"
+
+source:
+  url: https://github.com/SegataLab/metaclock/archive/{{ version }}.tar.gz
+  sha256: 146e6e8b5c8d62dc958a41cfb5a53cae9cc57bbc1da82d4020b217aafc180429
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv"
+
+requirements:
+  host:
+    - python =3.6.10
```

Comment:
Can you remove the pins where possible? Or at least make them a lot less strict? Otherwise it will cause a lot of problems resolving dependencies when the package is installed.